### PR TITLE
No unused vars as errors

### PR DESCRIPTION
**Pull request by webpro**
As I think unused variables are not just warnings, but errors.

### CODE REVIEW

**Comment by alber70g** [comment]
Does this now block the local build as well? Or is there a way to ignore that locally? It would be extremely annoying to be blocked by this during development. 

Let's say you want to try something out. You comment out a block of code, replace it with a `console.log`, and build and run it. Now you're faced with a blocking error that a variable is unused. That's undesirable

**Comment by webpro** [comment]
> Does this now block the local build as well? Or is there a way to ignore that locally? It would be extremely annoying to be blocked by this during development.

This is why linting and building should be separate. Then you can still build. But I see we're not there yet. So let's wait with merging this one until #612 is merged first.

**Review by sstraatemans — APPROVED** [review]

**Review by timoheddes — APPROVED** [review]

File: .changeset/gorgeous-ducks-explode.md
Comment (alber70g) [inline .changeset/gorgeous-ducks-explode.md:1]: When we merge this file like this. Will all subsequent changesets not trigger a "missing" changeset, and inadvertently result in missing changeset entries when we release?

**Review by alber70g — COMMENTED** [review]
One question about the changeset file

**Review by alber70g — APPROVED** [review]

**Pull request merged**
